eng/targets/Helix.Common.props

```diff
@@ -21,7 +21,7 @@
   <!-- x64 PR(ci.yaml) required queues for internal and public cases -->
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' != 'true'">
     <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Windows.11.Amd64.ClientPre.Open" Platform="Windows" />
+    <HelixAvailableTargetQueue Include="Windows.Amd64.Server2022.Open" Platform="Windows" />
```

Comment:
This queue does indeed exist but also the left one does. (the problem was that `Windows.11.Amd64.ClientPre` didn't get created)  Do you have some sort of thing that is stripping off .Open ?

Reply:
> Do you have some sort of thing that is stripping off .Open ?

Yes, we do that automatically in internal builds using a `$(_UseHelixOpenQueues)` variable in ci.yml